pkg/types/names.go

```diff
@@ -31,4 +31,5 @@ const (
 	NsxCertTempPath             string = "/tmp/nsx.cert"
 	NsxKeyTempPath              string = "/tmp/nsx.key"
 	NsxCATempPath               string = "/tmp/nsx.ca"
+	NsxNodeAgentContainerName   string = "nsx-node-agent"
```

Comment:
I pray we will never decide to change this name!

Reply:
lol. that's why it's a const ;)